packages/client-sdk-nodejs-compression/src/internal/compression/zstd-compression.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  MomentoLogger,
+  MomentoLoggerFactory,
+  CompressionLevel,
+  ICompression,
+} from '@gomomento/sdk';
+import * as zstd from '@mongodb-js/zstd';
+import {convert} from '@gomomento/sdk/dist/src/internal/utils';
+
+class ZtsdCompressor implements ICompression {
+  private readonly logger;
+  constructor(logger: MomentoLogger) {
+    this.logger = logger;
+  }
+
+  async compress(
+    compression: CompressionLevel,
+    value: Uint8Array
+  ): Promise<Uint8Array> {
+    let level;
+    switch (compression) {
+      case CompressionLevel.Default:
+        level = 3;
+        break;
+      case CompressionLevel.Fast:
+        level = 1;
+        break;
+      case CompressionLevel.Best:
```

Comment:
The word `Best` seems subjective/maybe wrong word to use? I don't have better word of top of my head but calling Best feels wrong... Maybe `Strongest`...?

Reply:
Inspiration comes from zstd librray which uses "Best" as a compression level.

Reply:
👍 ok

Reply:
@malandis Suggested using `SmallestSize` instead of `Best` which I like 👍